### PR TITLE
WRP-4228: Add `shadowed` prop to `Panels.Header` and `Button`

### PR DESCRIPTION
**Pull request by juwonjeong**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [x] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Add prop to enable shadow effects on buttons and text in `Panels.Header`.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Use text-shadow CSS property to add shadow to text.

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)
The UX guide only mentioned close button. But for consistency, I also applied a shadow effect to the back button (In Panels/Header.js, line 461).

### Links
[//]: # (Related issues, references)
WRP-4228

### CODE REVIEW

File: Button/tests/Button-specs.js [hunk 1]
```diff
@@ -198,6 +198,17 @@ describe('Button', () => {
 		});
 	});
 
+	describe('with shadowed', () => {
+		test('should have shadowed class', () => {
```
Comment (MikyungKim) [inline Button/tests/Button-specs.js:202]: same here.

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,8 +7,10 @@ The following is a curated list of changes in the Enact sandstone module, newest
 ### Added
 
 - `sandstone/Button` prop `roundBorder`, to make both sides of button fully rounded
+- `sandstone/Button` and `sandstone/Panels.Header` prop `shadowed` to add shadow to text and buttons
 
 ### Fixed
+
```
Comment (MikyungKim) [inline CHANGELOG.md:13]: :awthanks:

File: tests/screenshot/apps/components/Button.js [hunk 3]
```diff
@@ -140,6 +140,11 @@ const ButtonTests = [
 		component: <Button icon="plus" iconPosition="before" roundBorder size="large" />
 	},
 
+	// shadowed
+	...withConfig({wrapper: {light: true, padded: true}}, [
+		<Button shadowed backgroundOpacity="transparent" icon="plus" minWidth={false} />
```
Comment (MikyungKim) [inline tests/screenshot/apps/components/Button.js:145]: Could we omit `backgroundOpacity="transparent" here?

File: tests/screenshot/apps/components/Button.js [hunk 4]
```diff
@@ -244,7 +249,10 @@ const ButtonTests = [
 		{
 			textSize: 'large',
 			component: <Button icon="plus" iconPosition="before" roundBorder size="large" />
-		}
+		},
+
+		// shadowed + focused
+		<Button shadowed backgroundOpacity="transparent" icon="minus" minWidth={false} />
```
Comment (MikyungKim) [inline tests/screenshot/apps/components/Button.js:255]: same here.

**Review by MikyungKim — APPROVED** [review]
:lgtm:
Let's merge this after releasing 2.6.2.

**Review by 0x64 — APPROVED** [review]

**Pull request merged**